tests/robot-tests/tests/admin_and_public/bau/release_and_publication_role_permissions.robot

```diff
@@ -0,0 +1,358 @@
+    [Tags]  HappyPath    
+    user signs in as bau1
+    user goes to url  %{ADMIN_URL}/administration/users
+
+Navigate to manage users as bau1
+    [Tags]  HappyPath
+    user clicks link  Manage  xpath://td[text()="ees-analyst1@education.gov.uk"]/..
+    user waits until page does not contain loading spinner
+    
+Remove release owner access from Analyst1 
+    [Tags]  HappyPath
+    user waits until element is enabled  css:[data-testid="remove-release-role-Approver"]
+    user clicks element  css:[data-testid="remove-release-role-Approver"]
+
+Assign viewer only access to Analyst1
+    [Tags]  HappyPath
+    user waits until page does not contain loading spinner
+    user waits until element is enabled  css:[name="selectedReleaseId"]
+    user selects from list by label  css:[name="selectedReleaseId"]  ${RELEASE_NAME}
+
+    user waits until element is enabled  css:[name="selectedReleaseRole"]
+    user selects from list by label  css:[name="selectedReleaseRole"]  Viewer
+    user clicks button  Add release access
+    
+Sign in as Analyst1 User1 & navigate to publication
+    [Tags]  HappyPath
+    user signs in as analyst1
+    user selects theme and topic from admin dashboard  %{TEST_THEME_NAME}  %{TEST_TOPIC_NAME}
+    user waits until page contains accordion section   ${PUBLICATION_NAME}  120
+
+Navigate to release as a viewer
+    [Tags]  HappyPath
+    user selects theme and topic from admin dashboard  %{TEST_THEME_NAME}  %{TEST_TOPIC_NAME}
+    user waits until page contains accordion section   ${PUBLICATION_NAME}
+    user opens accordion section  ${PUBLICATION_NAME}
+
+    ${accordion}=  user gets accordion section content element   ${PUBLICATION_NAME}
+    ${details}=  user gets details content element  ${RELEASE_TYPE} (Live - Latest release)  ${accordion}
+
+    user waits until parent contains element   ${details}   xpath:.//a[text()="View this release"]
+    ${edit_button}=  get child element  ${details}  xpath:.//a[text()="View this release"]
+    user clicks element   ${edit_button}
+
+    user waits until h2 is visible  Release summary
+    user checks summary list contains   Publication title  ${PUBLICATION_NAME}
+
+Check release viewer cannot approve release 
+    [Tags]  HappyPath
+    user clicks link   Sign off
+    user waits until page does not contain loading spinner
+    user checks page does not contain  Edit release status
+
+Login as bau1 to remove viewer access 
+    [Tags]  HappyPath
+    user signs in as bau1
+    user goes to url  %{ADMIN_URL}/administration/users 
+
+Navigate to manage users page to remove viewer access
+    [Tags]  HappyPath    
+    user clicks link  Manage  xpath://td[text()="ees-analyst1@education.gov.uk"]/..
+    user waits until page does not contain loading spinner
+
+Give release contributor access to Analyst1
+    [Tags]  HappyPath
+    user scrolls to element  //*[text()="Analyst1 User1"]
+        
+    user scrolls to element  css:[name="selectedReleaseId"]
```

Comment:
I don't think you need these two one after the other.  I'd delete `user scrolls to element //*[text()="Analyst1 User1"]`